bcfs/src/lib.rs

```diff
@@ -1,7 +1,7 @@
 #![feature(bind_by_move_pattern_guards, cell_update)]
 
-#[cfg(feature = "ffi")]
-pub mod ffi;
+// #[cfg(feature = "ffi")]
```

Comment:
remove vs use?

Reply:
It's dead code at this point. I know that we'll want to use ffi in some form once we get to testing, but the interface is unstable enough that it's not clear what the ideal ffi should be. I'll just remove the files entirely since they're already in version control.